README.md

```diff
@@ -11,7 +11,7 @@ They are also used for each release of the following partner projects:
 - [u-track](http://downloads.openmicroscopy.org/u-track)
 - [OMERO.webtagging](http://downloads.openmicroscopy.org/webtagging)
 - [OMERO.searcher](http://downloads.openmicroscopy.org/searcher)
-- [OMERO.figure](http://downloads.openmicroscopy.org/figure)
+- [OMERO.figure](http://downloads.openmicroscopy.org/figure) (legacy)
```

Comment:
On this PR - yes, adding '(legacy version)' to the heading on the latest release page. On the index page, not that I can think of.